File: myHTML/hello.js
var calc = document.getElementById('calculate');
var result = document.getElementById('result');
calc.onclick = function () {
	var num1 = calc.form.num1.value;
	var num2 = calc.form.num2.value;
	result.textContent = num1 * num2;
	return false;
}

File: myHTML/hello.py
#!/usr/bin/python

from flask import Flask, request, render_template

app = Flask(__name__)

@app.route('/')
def hello():
	try:
		num1_string = request.args.get('num1')
		num2_string = request.args.get('num2')
		number1 = (float(num1_string) if '.' in num1_string else
			int(num1_string))
		number2 = (float(num2_string) if '.' in num2_string else
			int(num2_string))
		result = number1 * number2
		return render_template('hello.html',
			num1 = number1, num2 = number2, result = result)
	except:
		return render_template('hello.html')
		
	if __name__ == '__main__':
		app.debug = True
		app.run(host = '0.0.0.0')